admin/views.py

```diff
@@ -495,8 +511,16 @@ def resource_create(resource_type):
     attributes_to_save = {}
     for attribute in editable_attributes:
         attribute_value = request.form.get(attribute["name"])
-        validated_attribute_value = validate_resource_attribute(resource_type, attribute, attribute_value)
-        attributes_to_save[attribute['name']] = validated_attribute_value
+        validated_attribute_value = validate_resource_attribute(
+            resource_type, attribute, attribute_value
+        )
+        attributes_to_save[attribute["name"]] = validated_attribute_value
+
+        if attribute["name"] == "password":
```

Comment:
we have a configurable column name in `admin_view.py` within `admin_configs` variable for the identifier secret. Let's use that here. So essentially it will look something like this:
```py
if attribute["name"] == admin_configs['user']['secret']:
```